candidateport.py

```diff
@@ -0,0 +1,28 @@
+class Candidate:
+   cCount = 0
+   def __init__(self, c_name, c_id, c_aadhar):
+      self.c_name = c_name
+      self.c_id = c_id
+      self.c_aadhar = c_aadhar
+      Candidate.cCount += 1
+      global cd_id,cd_name
+      cd_id=c_id
+      cd_name=c_name
+      cdict={c_name:c_id}
+   def displayCount(self):
+     print "Total Candidates are: %d" % Candidate.cCount
+
+   def displayCandidate(self):
+      print "Name : ", self.c_name,  ", Voter Id ", self.c_id,  ", Aadhar Id: ", self.c_aadhar
+   def cdadd(self):
```

Comment:
I didn't understand the method name here.  Name it in more readable fashion.

This method should not be reading console.

Reply:
Same goes with voter.